worker-archetype/src/main/resources/archetype-resources/__rootArtifactId__-container/pom.xml

```diff
@@ -34,7 +34,6 @@
     <properties>
         <maven.install.skip>true</maven.install.skip>
         <maven.deploy.skip>true</maven.deploy.skip>
-        <docker.maven.version>0.13.6</docker.maven.version>
```

Comment:
Are these archetype updates/fixes required for this Jira?  If not then remove them from this PR and put them in a separate one.

Reply:
With the change made for this jira, the tests fail without the healthcheck being called. The changes in the archetype are to get the container to call the healthcheck.